usecases/mail-sender/src/main/java/com/oracle/cloud/sdk/samples/MailSenderController.java

```diff
@@ -0,0 +1,37 @@
+package com.oracle.cloud.sdk.samples;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class MailSenderController {
+	
+	@Autowired
+	MailSenderService service;
+	
+	@RequestMapping(value="/hello")
+	public String hello() {
+		return "Hello World";
+	}
+	
+	@RequestMapping(value="/send", method = RequestMethod.GET)
+	public String send() {
+		service.sendMail(new MailSenderService.MailContent("<YOUR_MAIL_ID>", "Hello from OCI SpringBoot", "Test Message"));
+		return "Done";
+	}
+	
+	@RequestMapping(
+	    value = { "/send" }, 
+	    method = RequestMethod.POST, 
+	    produces = "application/json", 
+	    consumes = "application/json")
+	@ResponseBody
+	public String sendPost(@RequestBody MailSenderService.MailContent content) {
+		service.sendMail(content);
+		return "Done";
```

Comment:
Is there any error handling that might fit well here?  Usually when a request is sent, it's possible that it could fail (or return an error response code).  This looks like it always returns "`Done`", but might at least look at the response code (at minimum look for a 2xx) and handle other codes as an error?

Reply:
Fixed. now it would show failure message with appropriate error code as below.

`
{
    "timestamp": "2023-02-10T01:24:25.777+00:00",
    "status": 500,
    "error": "Internal Server Error",
    "path": "/send"
}
`